service/src/service.rs

```diff
@@ -133,7 +133,7 @@ pub async fn run() -> anyhow::Result<()> {
         MainConfig::parse(indexer_config::ConfigPrefix::Service, &cli.config).map_err(|e| {
             error!(
                 "Invalid configuration file `{}`: {}",
-                cli.config.display(),
+                cli.config.unwrap_or_default().display(),
                 e
             );
             anyhow!(e)
```

Comment:
In case there's a missing config field while parsing the config file, I'd like to let the user know that it's possible to use the `--config`. Please also update `tap-agent` accordingly.